Tests/MessagingInApp/Extensions/GistExtensions.swift

```diff
@@ -20,10 +20,10 @@ extension Message {
     }
 
     static var random: Message {
-        Message(messageId: .random, campaignId: .random)
+        Message(elementId: nil)
     }
 
     static var randomInline: Message {
-        Message(messageId: .random, campaignId: .random, elementId: .random)
+        Message(elementId: .random)
     }
```

Comment:
This was only changed out of convenience. The `Message` constructor used in tests has default values for all parameters so some parameters are no longer needed.